src/main/java/io/stargate/sgv2/jsonapi/api/v1/CollectionResource.java

```diff
@@ -181,7 +181,14 @@ else if (error instanceof JsonApiException jsonApiException) {
                       embeddingProviderFactory.getConfiguration(
                           dataApiRequestInfo.getTenantId(),
                           collectionProperty.vectorConfig().vectorizeConfig().provider(),
-                          collectionProperty.vectorConfig().vectorizeConfig().modelName());
+                          collectionProperty.vectorConfig().vectorizeConfig().modelName(),
```

Comment:
nit: I know use of tmp variables is frowned upon, but here it might make sense to use:

```
var vectorizeConfig = collectionProperty.vectorConfig().vectoriConfig();
```

to shorten code to be more readable